docker-compose.yaml

```diff
@@ -24,3 +24,11 @@ services:
       always
     depends_on:
       - docker
+
+  db:
+    image: 'postgres:latest'
+    volumes:
+      - ./postgres_data:/var/lib/postgresql/data
+    env_file: .env
```

Comment:
seems like environment variables for [official postgres](https://hub.docker.com/_/postgres) image and env vars that we use in our app differ

Reply:
now our app will fail to launch ((